src/queries/voting-data.ts

```diff
@@ -20,45 +20,50 @@ const aragonEnv = {
  * @param id - id of the vote to query (I.E. 21)
  * @param votingContractAddress - voting contract to query -- defaults to the main sarcophagus DAO address
  */
-function formatVoteId(id: string | number, votingContractAddress: string = '0xf483c1f7858dd19915d0689d26cb3487fc90b640'): any {
-  const aragonVoteIdPrefix = `appAddress:${votingContractAddress}-vote:`
-  const aragonVoteId = aragonVoteIdPrefix + "0x" + Number(Number(id).toString(16)).toString(16)
-  return aragonVoteId
+function formatVoteId(
+  id: string | number,
+  votingContractAddress: string = '0xf483c1f7858dd19915d0689d26cb3487fc90b640'
+): any {
+  const aragonVoteIdPrefix = `appAddress:${votingContractAddress}-vote:`;
+  const aragonVoteId = aragonVoteIdPrefix + '0x' + Number(Number(id).toString(16)).toString(16);
+  return aragonVoteId;
 }
 
+/**
+ * Retrieves details of vote including address of voters and timing of execution
+ *
+ * @param web3 - blockchain provider
+ * @param voteId - id of the vote to query (I.E. 21)
+ */
```

Comment:
The `web3` argument should be typed as `Web3`. If you have auto import set up, use that. If not, `import Web3 from "web3".

Reply:
There are certainly cases where using `any` is appropriate, but think long and hard before doing so. It completely defeats the purpose of using typescript. 99% of the time you should not use `any`. It takes time to figure out what types to use in some cases but it's a good use of time.